maquetado/scss/detail.scss

```diff
@@ -1,29 +1,51 @@
 @use 'colors' as colors;
+@use 'sizes' as sizes;
 
-.book-detail {
+.book-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
   max-width: 100%;
   padding: 10px;
-  width: 600px;
+  width: 900px;
 
-  .book-title {
-    font-size: 32px;
-    font-weight: bold;
-    padding: 10px 0;
+  .book-cover {
+    max-width: 100%;
+    width: 261px;
   }
-  .book-genre {
-    @include colors.secondary-text-color;
-    font-size: 24px;
-  }
-  .separator {
-    border: none;
-    border-top: 4px solid colors.$earls-green;
-  }
-  .book-prop {
-    font-size: 20px;
-    margin-top: 30px;
-    & > .book-prop-description {
+  .book-detail {
+    max-width: 100%;
+    padding-left: 52px;
+    width: 600px;
+
+    .book-title {
+      font-size: 32px;
+      font-weight: bold;
+      padding-bottom: 10px;
```

Comment:
Remember we use margin to separate from other elements and padding just to separate from borders. So, if you want more space between the element and the bottom of the card, use a margin instead.